gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Wed Jan 09 07:53:43 PST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-bin.zip
```

Comment:
This should be rolled back. Gradle requires that plugins are built against the minimum version of Gradle that they support.

Reply:
Ok didn't know that. Reverted to `4.10.3`.